layershellev/src/lib.rs

```diff
@@ -1824,6 +1826,13 @@ impl<T: 'static> WindowState<T> {
                         None,
                     ));
                 }
+                Some(ReturnData::RequestCompositor) => {
```

Comment:
Emm. I mean add another return value like `RequestSetRegion`... and it should not be in this place I think